scripts/generate-packages.ps1

```diff
@@ -13,9 +13,17 @@
 # Due to this, when calling `choco pack` the version should not contain the build number (9.0.0-rc).
 # At the same time the the url inside the ps1 file that downloads the scanner should be correct and contain the build number.
 
+[CmdletBinding()]
```

Comment:
What does this do?
Do we need it?

Reply:
This does multiple things. It transforms the function into an advanced one that adds support for parameter attributes, common parameters like `-Verbose` and `-Debug,` lifecycle methods, and more. We don't use all these features now but I prefer to keep the property.